acquia_lift.install

```diff
@@ -203,6 +203,32 @@ function _acquia_lift_build_menu($menu_name) {
   );
   menu_link_save($item);
 
+  // Create a link to the all campaigns page.
+  $item = array(
+    'link_title' => 'All campaigns',
+    'link_path' => 'admin/structure/personalize',
+    'menu_name' => $menu_name,
+    'options' => array(
+      'attributes' => array(
+        'aria-role' => 'button',
+        'aria-pressed' => 'false',
+        'class' => array(
+          'acquia-lift-campaign-all',
```

Comment:
Do the individual acquia-lift-campaign-all, acquia-lift-option-sets-all, and acquia-lift-goals-all serve a particular purpose?  I don't see them referenced again.